### PR TITLE
 Expose City Geo Name ID to IpLocation result (closes snowplow#129)

### PR DESCRIPTION
**Pull request by nitayk**

### CODE REVIEW

File: src/main/scala/com.snowplowanalytics.maxmind.iplookups/model.scala [hunk 1]
```diff
@@ -13,38 +13,44 @@
...
+  final case class ParsedSubdivision(name: String, isoCode: String, geoNameId: Long)
+
   /** A case class wrapper around the MaxMind CityResponse class. */
   final case class IpLocation(
     countryCode: String,
     countryName: String,
     region: Option[String],
+    regionName: Option[String],
     city: Option[String],
+    cityGeoNameId: Option[Long],
```
Comment (nitayk) [inline src/main/scala/com.snowplowanalytics.maxmind.iplookups/model.scala:40]: Exposing `Geo Name ID`

File: src/main/scala/com.snowplowanalytics.maxmind.iplookups/model.scala [hunk 2]
```diff
@@ -13,38 +13,44 @@
...
     latitude: Float,
     longitude: Float,
     timezone: Option[String],
     postalCode: Option[String],
     metroCode: Option[Int],
-    regionName: Option[String],
     isInEuropeanUnion: Boolean,
     continent: String,
-    accuracyRadius: Int
+    accuracyRadius: Int,
```
Comment (nitayk) [inline src/main/scala/com.snowplowanalytics.maxmind.iplookups/model.scala:48]: Exposing other metadata

File: src/main/scala/com.snowplowanalytics.maxmind.iplookups/model.scala [hunk 3]
```diff
@@ -68,25 +74,37 @@ object model {
...
-      } catch {
-        case _: NoSuchMethodError => false
-      }
+      val isInEuropeanUnion =
+        try cityResponse.getCountry.isInEuropeanUnion
+        catch {
+          case _: NoSuchMethodError => false
+        }
+
+      val geoNameId: Long = cityResponse.getCity.getGeoNameId
```
Comment (nitayk) [inline src/main/scala/com.snowplowanalytics.maxmind.iplookups/model.scala:83]: Explicit type to avoid `Long2Long` WARN

**Comment by nitayk** [comment]
It seems develop branch is not up-to-date with master

**Comment by hagaiovadia** [comment]
@alexanderdean can you take a look?

**Comment by hagaiovadia** [comment]
@oguzhanunlu @dilyand ?

**Pull request closed without merging**